go/stellar/wallet_state.go

```diff
@@ -107,6 +107,46 @@ func (w *WalletState) BaseFee(mctx libkb.MetaContext) uint64 {
 	return w.options.BaseFee(mctx, w)
 }
 
+// AccountName returns the name for an account.
+func (w *WalletState) AccountName(accountID stellar1.AccountID) (string, error) {
```

Comment:
so these are only used in tests right now? will they be used in other code as well?

Reply:
Just in tests, but seems fine if they are needed elsewhere.  Currently they aren't...